pages/api/hello.js

```diff
@@ -0,0 +1,6 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
Do we need this file at all? If not let's blast it away

Reply:
Hmm - I'm not sure. It was part of the scaffolding from the create command. Comment all the code in it doesn't seem to break anything. I'll look into this.

Reply:
I think this is just an example they include to demonstrate routing. Deleting.